src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.PureComponent {
+  render() {
+    return (
+      <div className="App">
```

Comment:
- [ ] [Optional] [Highly recommended] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=the%20next%20project.-,Use%20functional%20components,-Create%20at%20least), Please make the App component a functional component again.

### ⚠️I marked this as optional because the last reviewer didn't pay attention to the new requirement for this particular project so please consider doing the change⚠️

Reply:
Thanks for your suggestion.